libzeth/test/note_test.cpp

```diff
@@ -188,8 +188,10 @@ TEST(TestNoteCircuits, TestOutputNoteGadget)
     // inner_k = blake2sCompress(a_pk || rho)
     // outer_k = blake2sCompress(r || [inner_commitment]_128)
     // cm = blake2sCompress(outer_k || 0^192 || value_v)
-    bits256 cm_bits256 = get_bits256_from_vector(hex_digest_to_binary_vector(
-        "626876b3e2747325f469df067b1f86c8474ffe85e97f56f273c5798dcfccd925"));
+    // bits256 cm_bits256 = get_bits256_from_vector(hex_digest_to_binary_vector(
+    //    "626876b3e2747325f469df067b1f86c8474ffe85e97f56f273c5798dcfccd925"));
```

Comment:
Same as above, let's either uncomment and use the var, or just remove the comment

Reply:
Updated.